client/me/security-2fa-backup-codes-list/index.jsx

```diff
@@ -91,11 +76,11 @@ class Security2faBackupCodesList extends Component {
 		saveAs( toSave, `${ this.props.username }-backup-codes.txt` );
 	};
 
-	getBackupCodePlainText( backupCodes ) {
+	getBackupCodePlainText = ( backupCodes = this.props.backupCodes ) => {
```

Comment:
The `backupCodes` param is never used, we could as well use `this.props.backupCodes` in the function body.